app/views/profiles/_attendances_details.html.erb

```diff
@@ -11,11 +11,17 @@
     </div>
 
     <ul class="<%= status %> attendees-list">
-      <% attendances.map(&:user).each do |user| %>
+      <% attendances.each do |attendance| %>
+        <% user = attendance.user %>
```

Comment:
Typically I would just wrap the whole block into a partial and pass it a local, `user`. In this case, probably something like `_attendances_details_user_detail`. Autocomplete yo!